ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyManager.java

```diff
@@ -352,6 +352,7 @@ public void manageChildren(
             arrayContains(tagsToDelete, viewToRemove.getId())) {
           // The view will be removed and dropped by the 'delete' layout animation
           // instead, so do nothing
```

Comment:
This comment probably needs to be updated.  Since do nothing now looks like doing something.